TelCo.ColorCoder/ColorManualTester.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{  
+	class ColorManualTester : IColorManual

```

Comment:
The test and the actual code have parallel implementations. This means the test can pass, while the actual implementation could have issues on the customer side. When the test doesn't give confidence, it loses its utility. Why not bring the common parts between the test and the actual into a function, call it in both places? We can check the result in test code and print the result in production code. The disparity between test and production code reduces.